finesse/config.py

```diff
@@ -80,3 +80,6 @@
 
 TEMPERATURE_MONITOR_COLD_BB_IDX = 7
 """Position of the cold blackbody on the temperature monitoring device."""
+
+TC4820_MAX_POWER = 511
+"""The maximum value for the power property."""
```

Comment:
What does this value mean? I'm guessing it's just a proportion rather than a physical unit?

How does Jon know this is the right maximum?

Reply:
Not sure what the units are, the manual states "511 represent 100% output". (Note you had this in `finesse/hardware/temperature/tc4820.py` but it was unused - I've just moved it to here so we can use it in `finesse/gui/temp_control.py`.)

Reply:
Okey doke. Whoops, it looks like I knew what it meant at some point then forgot and assumed it was 0-100 :laughing:

Reply:
I wonder if this will also be needed in the CSV-writing routine - I think we want the power output as a percentage of maximum, so we'll need to divide by 5.11.